### ISSUE TITLE
Blind trait has so much dead space

### ISSUE THREAD

**Issue by Stealthbomber16** [space-wizards/space-station-14#11632]
Ideally, the blindness trait should zoom in your camera so that you dont like 8 miles of blank space around your character- especially on higher resolutions.
![image](https://user-images.githubusercontent.com/100171035/193358672-8cedcee4-42f2-4f32-879b-938cdc657fde.png)

I propose making the blindness circle larger on the blind trait, and significantly lowering the eye zoom component of whoever has the trait.

**Comment by Elijahrane** [comment]
Should these improvements apply to blindness in general?

**Comment by EmoGarbage404** [comment]
I think thatd make sense. Extend the circle to just about half of what your interaction range is and probably make it a soft fade rather than a hard circle.

**Comment by EmoGarbage404** [comment]
As for eye zoom, i feel like thatd look weird.

**Comment by SpliTistiC** [comment]
methinks BlindnessTrait should have more vision than someone who is just wearing a blindfold, and maybe the fact that BlindnessTrait shouldn't get slowdown from flashes either would be cool.

**Comment by Elijahrane** [comment]
> As for eye zoom, i feel like thatd look weird.

having messed with eye zoom quite a lot it looks pretty alright. All pixels are the same scale relative to each other, just bigger

**Comment by SpliTistiC** [comment]
blind trait giving a zoom will further reduce any sense of direction- like people speaking around them and it just looks really odd in consistency

**Comment by Kromabae** [comment]
just so you know by the way, blindness does give nightvision in the small area you actually can see

**Comment by SpliTistiC** [comment]
well yeah- can't see light and all- but zooming in would be hella jarring- nothing else does that in the game- whether you're dragon or mouse- even if blind methinks

**Comment by Stealthbomber16** [comment]
My main issue is that I'm having the problem of having an extremely small amount of space actually be used on my screen. I get that it's blindness, but that space has to be used for something.

**Comment by moonheart08** [comment]
The zoom in would make blindness **even worse** by hiding where people are speaking.

**Comment by Stealthbomber16** [comment]
> The zoom in would make blindness **even worse** by hiding where people are speaking.

Can we make the FOV a cone then?

**Comment by moonheart08** [comment]
> > The zoom in would make blindness **even worse** by hiding where people are speaking.
> 
> Can we make the FOV a cone then?

I frankly think blindness is fine where it is.
You're not going to magically procure content to fill a dark void when you literally cannot see anything.

**Comment by metalgearsloth** [comment]
Could just find a shader to use.

**Comment by moonheart08** [comment]
> Could just find a shader to use.

giving blind folk More Visibility also doesn't reaaallly make sense?

**Comment by metalgearsloth** [comment]
It doesn't need to change visibility, just need to add some pattern to black space.

**Comment by SpliTistiC** [comment]
1. Methinks the blind trait could be more FUN while still being BAD by increasing their little vision by a tad more- and maybe a cool shader to boot (but make it balanced that BLINDtrait people can "see" more than NORMAL people who are just wearing a blindfold. (they're experts)
2. A cool thing in TG station is blindtrait has more range- a cool shader, and a neat balance or (fun vs realism) thing where every X amount of seconds (quite long) the sceen flickers revealing the screen as if you had normal vision for half a second. At the end of the day I'm only advocating for more fun to be had, and I a person who uses the blind trait could enjoy it more with even just a tad more sense range

**Comment by BasedUser** [comment]
in real life blind people can sense being in a lit area

**Comment by SpliTistiC** [comment]
We could probably get things like myopia where you start with screen blurrines (need glasses) but thats another discussion- on a real note regarding the discussion in #ideaguys and #ideamen- all thats needed RIGHT NOW is an increase in Blindtrait range compared to blindfold range, they SHOULD BE 2 different things- I see zero issue in this and it makes it more fun for the people who choose this trait- at the end of the day we're spicing up our playstyle or having different perspectives- why not improve it.

**Comment by SpliTistiC** [comment]
did you read what I said?

**Comment by Nopey** [comment]
Should be resolved by #23212

**Comment by UbaserB** [comment]
> Should be resolved by #23212

no, when the accessibility setting is turned on there is still a lot of dead space.

**Comment by moonheart08** [comment]
> > Should be resolved by #23212
> 
> no, when the accessibility setting is turned on there is still a lot of dead space.

At that point, it is intended behavior. We can't really fix it.
